src/renderers/shaders/ShaderChunk/clipping_planes_vertex.glsl

```diff
@@ -0,0 +1,4 @@
+#if NUM_USER_CLIPPING_PLANES > 0 && ! defined( STANDARD ) && ! defined( PHONG )
+	vViewPosition = - mvPosition.xyz;
```

Comment:
We maybe should just always pass world position into the fragment shader?  It is just a single and useful uniform.  That or have a way to reconstruct it.  Seems wasteful/complex that we maintain so many if tests to save a single very common uniform from being passed.